### ISSUE TITLE
[Feature Request] Datatable elements have insufficient color contrast

### ISSUE THREAD

**Issue by plweil** [vuetifyjs/vuetify#6647]
### Problem to solve
Several datatable elements (light theme) lack sufficient color contrast to meet WCAG accessibility guidelines. These will be difficult or impossible to see for many users. Examples include: the arrows for sorting columns; the footer text (rows per page, etc.). Increasing the color contrast will make these visible to all (or at least many more) sighted users.

### Proposed solution
Although this is officially a feature request, I really consider this a bug. Solution: darken the colors of these elements and test them in an accessibility testing tool like axe until no color contrast issues remain.

**Comment by sh7dm** [comment]
@plweil it looks fine for me:
![image](https://user-images.githubusercontent.com/19504461/58851444-77d4d300-869b-11e9-842b-a77cf8ca6646.png)

**Comment by sh7dm** [comment]
where are issues located?

**Comment by plweil** [comment]
As mentioned above: the up/down arrows for sorting columns (2.68:1). Items in the footer row; e.g., Rows per page (and other, similar items in the footer row) (3.36:1). Using the light theme.

**Comment by 06b** [comment]
When running Accessibility Insights for Web on the Datatable I was able to get the following result when hovering over the a column for the sorting arrow to appear.

![image](https://user-images.githubusercontent.com/1302542/58881992-51b62f80-86a9-11e9-97a5-b972f0037820.png)

Hiding the failure exclamation mark:
![image](https://user-images.githubusercontent.com/1302542/58881959-419e5000-86a9-11e9-9056-c3032944cfce.png)

Title: WCAG 1.4.3: Elements must have sufficient color contrast (i)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI)

Target application title: Vuetify Example Pen
Target application url: https://codepen.io/anon/pen/oRVmdo?&editable=true&editors=101

Element path: #result;.sortable.text-xs-left[aria-sort="none"]:nth-child(5) > i

Snippet: <i aria-hidden="true" class="v-icon material-icons theme--light" style="font-size: 16px;">arrow_upward</i>

How to fix:
Fix any of the following:
  Element has insufficient color contrast of 1.72 (foreground color: #c5c5c5, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

Environment:
Chrome version 74.0.3729.169

====

This accessibility issue was found using Accessibility Insights for Web 2.4.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by KaelWD** [comment]
The inactive sort arrow is 38% black like the spec says, but it also has 60% opacity for some reason, making it lighter than it should be.

**Comment by sh7dm** [comment]
@KaelWD I'll remove opacity from it. Gimme some time

**Comment by tjeb** [comment]
The contrast of the 'no data available' message is also too low without a manual override, btw. As is the contrast of the default 'loading' message.

**Comment by johnleider** [comment]
This is resolved as of v3.

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
